tests/platform/windows/windows-chromeSST-testSpec.js

```diff
@@ -0,0 +1,149 @@
+/*
+GPII Integration and Acceptance Testing
+
+Copyright 2014 Emergya
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+"use strict";
+var fluid = require("universal"),
+    gpii = fluid.registerNamespace("gpii");
+
+gpii.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.windows.chromeSST");
+
+
+
+gpii.tests.windows.chromeSST = [
+    {
+        name: "Acceptance test for SST translate service in Chrome",
+        userToken: "chrome_sst_translate",
```

Comment:
As i mentioned in the email, the NP sets are missing for the tests -- namely chrome_sst_translate, chrome_sst_fontconverter, chrome_sst_callwebanywhere and chrome_sst_combined .. which should all be located in the testData/preferences/acceptanceTests folder.